src/EscapeFormula.php

```diff
@@ -112,7 +112,7 @@ public function escapeRecord(array $record): array
      *
      * @param int|float|string|object|resource|array $cell the content of the cell
      *
-     * @return mixed the escaped content
+     * @return int|float|string|object|resource|array the escaped content
```

Comment:
I just now saw your test on this and reverted the change. 
Does it make sense to allow so many data types here. Maybe you should normalize the values?

Reply:
> I just now saw your test on this and reverted the change.
Does it make sense to allow so many data types here. Maybe you should normalize the values?

Yes it does in this context because of the way PHP generates CSV